examples/plot_nilearn_101.py

```diff
@@ -10,32 +10,38 @@
 import os
 
 #########################################################################
-# Let us use a Nifti file that is shipped with nilearn
+# Let us use an in-memory Nifti file that is shipped with nilearn
```

Comment:
I am not sure why this is written "in-memory Nifti file", given that we are using a filename below.